patternfly/patternfly-elements
# fix: pfe clipboard bug fixes

## Component name

pfe-clipboard


### Preview


Link(s) to demo page(s) where this element can be viewed:
- [Link](https://deploy-preview-1310--happy-galileo-ea79c4.netlify.app/elements/pfe-clipboard/demo/) 


### What has changed and why

- Fixed copyURLToClipboard() API example in Readme
- Added `--pfe-clipboard--FontSize` to the `text--success` slot.  I tested this with a tailwind.css example and it was showing different font-sizes for the `text` and `text--success` slots.

### Bad
![Screen Shot 2021-01-27 at 7 06 11 PM](https://user-images.githubusercontent.com/3428964/106070652-dbaed980-60d2-11eb-9461-339712a3b8e5.png)

### Good
![Screen Shot 2021-01-27 at 6 14 01 PM](https://user-images.githubusercontent.com/3428964/106070610-c5a11900-60d2-11eb-8e53-cdd239a29da1.png)




### Testing instructions



1. Visit https://deploy-preview-1310--happy-galileo-ea79c4.netlify.app/elements/pfe-clipboard/demo/
2. From the dev tools inspector, select a `pfe-clipboard` element
3. Add a new Element Style `--pfe-clipboard--FontSize: 2em`
4. Verify that the `Copy URL` is 2em
5. Add a new `copied` attribute to the element
6. the text--success text should be 2em

**Screenshots**
![Screen Shot 2021-01-28 at 2 46 53 PM](https://user-images.githubusercontent.com/3428964/106190798-25053480-6178-11eb-8bd2-a2f5dfcdc4b7.png)

![Screen Shot 2021-01-28 at 2 47 20 PM](https://user-images.githubusercontent.com/3428964/106190804-26cef800-6178-11eb-8c8d-64195e5bdf72.png)



#### Browser requirements

Your component should work in all of the following environments:



### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable.



### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
